app/src/main/java/org/breezyweather/sources/gadgetbridge/GadgetbridgeService.kt

```diff
@@ -122,6 +122,14 @@ class GadgetbridgeService @Inject constructor() : BroadcastSource {
                 maxTemp = day.day?.temperature?.temperature?.roundCelsiusToKelvin(),
                 minTemp = day.night?.temperature?.temperature?.roundCelsiusToKelvin(),
 
+                // TODO: humidity
+                windSpeed = day.day?.wind?.speed?.let {
```

Comment:
That's the wind speed only for the daytime period

Reply:
Interesting - that's what is being used for the current day as well, and now I'm not sure it should.

I don't see a field for "day" wind speed - would the average make sense?

Reply:
No, you should take the maximum of the two values, sometimes it will be daytime, sometimes nighttime. Accordingly, the wind direction must be from the highest wind speed of the two

Reply:
> Interesting - that's what is being used for the current day as well, and now I'm not sure it should.

I misread this - it was not.

Addressed.